.circleci/config.yml

```diff
@@ -44,16 +44,38 @@ commands:
           command: |
             wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
             sudo apt install ./google-chrome-stable_current_amd64.deb
+  install-xtensa-toolchain:
+    parameters:
+      variant:
+        type: string
+    steps:
+      # Cache the file because the Espressif download website is not particularly fast.
```

Comment:
Another option is to download the same file from GitHub Releases, which should be pretty fast: https://github.com/espressif/crosstool-NG/releases/tag/esp-2020r2

Reply:
Good idea, thanks!

Reply:
I've updated the PR with this. That makes the CI file smaller and simpler (no caches needed).